base/sysinfo.jl

```diff
@@ -98,7 +98,19 @@ Standard word size on the current machine, in bits.
 """
 const WORD_SIZE = Core.sizeof(Int) * 8
 
-global SC_CLK_TCK::Clong, CPU_NAME::String, JIT::String
+global SC_CLK_TCK::Clong
+"""
+    Sys.CPU_NAME::String
+
+A string representing the name of CPU.
```

Comment:
Would be good to give an example or two

e.g. on my machine it is `"skylake"` for the [Intel Core "Skylake" CPU](https://en.wikipedia.org/wiki/Skylake_(microarchitecture)).   (Wikipedia links in the docstrings are fine!)